datafusion/core/tests/parquet_filter_pushdown.rs

```diff
@@ -225,6 +230,77 @@ async fn single_file() {
         .run()
         .await;
 }
+
+#[cfg(not(target_family = "windows"))]
+#[tokio::test]
+async fn single_file_small_data_pages() {
```

Comment:
Here is the new test -- I verified manually the layout was good (typically makes 6 data pages for each column chunk)

Reply:
👍 cool!